modules/nf-core/velocyto/main.nf

```diff
@@ -0,0 +1,49 @@
+process VELOCYTO {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/velocyto.py:0.17.17--py38h24c8ff8_6':
+        'biocontainers/velocyto.py:0.17.17--py38h24c8ff8_6' }"
+
+    stageInMode 'copy'
+
+    input:
+    tuple val(meta), path(barcodes), path(bam), path(sorted_bam)
+    path gtf
+
+    output:
+    tuple val(meta), path("*.loom"), path("*.velocyto.log"), emit: loom
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    velocyto run $args -e ${meta.id} -b ${barcodes} -o . ${bam} ${gtf}
+
+    cp .command.out ${prefix}.velocyto.log
```

Comment:
That seems somewhat unconventional to me. I think 
```
velocyto run $args -e ${meta.id} -b ${barcodes} -o . ${bam} ${gtf} > ${prefix}.velocyto.log
```
or something like that, would resemble more what I see in other modules.